Client/sdk/game/CModelInfo.h

```diff
@@ -247,6 +297,18 @@ class CModelInfo
     // Vehicle towing functions
     virtual bool IsTowableBy(CModelInfo* towingModel) = 0;
 
+    // 2dfx functions
+    virtual C2DEffectSAInterface* Add2DFXEffect(const CVector& position, const e2dEffectType& type) = 0;
+    virtual void                  Remove2DFX(C2DEffectSAInterface* effect, bool isCustom = false) = 0;
+    virtual bool                  Remove2DFXEffectAtIndex(std::uint32_t index, bool includeDefault = false) = 0;
+    virtual bool                  RemoveAll2DFXEffects(bool includeDefault = false) = 0;
+
+    virtual C2DEffectSAInterface* Get2DFXFromIndex(std::uint32_t index) = 0;
```

Comment:
const?

Reply:
Const can only be GetCount because GetFromIndex changes the value of m_pInterface